src/engine/sysDef.cpp

```diff
@@ -798,7 +798,7 @@ void DivEngine::registerSystems() {
 
   sysDefs[DIV_SYSTEM_AMIGA]=new DivSysDef(
     "Amiga", NULL, 0x81, 0, 4, false, true, 0, false, 1U<<DIV_SAMPLE_DEPTH_8BIT,
-    "a computer from the '80's with full sampling capabilities, giving it a sound ahead of its time.",
+    "a computer from the '80's with full sampling capabilities, giving it a sound ahead of its time.\n\n- the maximum rate for sample playback is technically 31469Hz but anything higher than 28867Hz will sound glitchy on hardware.\n- sample lengths and loop will be set to an even number.\n- samples can't be longer than 131070.",
```

Comment:
I honestly think this description is a bit too long. We'll discuss this soon but I can approve the other changes for now.

Reply:
I've pulled it for now. We can revisit the idea later.